### DISCUSSION TITLE
Unfollowed users still showing up in "Following" feed

### DISCUSSION THREAD

**Discussion by ghost** [community/community#32784] [Other Feature Feedback, Questions, & Ideas]
Hello,

I followed an account for a small window of time, then unfollowed it, but its recent activity is still showing up in my "Following" feed. I have tried to follow/unfollow it to see if it would get it to stop, but it made no difference. I have confirmed it is new activity showing up, not just the activity I saw from the time I was actually following the account.

**Comment by minirop** [comment]
4 years later and it's still an issue?
